pkg/util/azureclient/mgmt/compute/virtualmachinescalesetvms_addons.go

```diff
@@ -0,0 +1,41 @@
+package compute
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-03-01/compute"
+)
+
+type VirtualMachineScaleSetVMsClientAddons interface {
+	ListVMs(ctx context.Context, resourceGroupName string, virtualMachineScaleSetName string, filter string, selectParameter string, expand string) ([]compute.VirtualMachineScaleSetVM, error)
```

Comment:
s/ListVMs/List/

Reply:
The list already exists, and it returns paginated results. I wanted one where without pagination involved in the end result